src/js/zqd/zqd.js

```diff
@@ -78,33 +79,29 @@ function zqdCommand(): string {
   return zqdBin
 }
 
-function zeekRunnerCommand(): string {
+function zeekRunnerCommand(zeekRunnerPref: string): string {
   const plat = platformDefs[process.platform]
   if (!plat) {
     throw new Error("unsupported platform for zqd")
   }
 
-  let zeekRunner = process.env.BRIM_ZEEK_RUNNER
-  if (!zeekRunner) {
-    // TODO: https://github.com/brimsec/brim/issues/741
-    // If the environment varible isn't set, allow the user to set
-    // a preferred zeekrunner location via preferences.
-    zeekRunner = resolve(join(zdepsDirectory, "zeek", plat.zeekRunnerBin))
-  }
-
-  if (!pathExistsSync(zeekRunner)) {
-    throw new Error("zeek runner not present at " + zeekRunner)
```

Comment:
I removed this thow since the value is now user specified. If they provided invalid input, the app would not start.

Instead, for now, if there is a bad zeekrunner value set, the app starts up fine, but we display an error when they try to import a pcap.

Here is what the error will look like if the zeek runner does not exist:

<img width="509" alt="Screen Shot 2020-05-12 at 4 11 34 PM" src="https://user-images.githubusercontent.com/3460638/81755884-e951ff80-946e-11ea-846a-0d5e8e81ad9c.png">

And here is if the zeek runner file is present but invalid

<img width="503" alt="Screen Shot 2020-05-12 at 4 14 25 PM" src="https://user-images.githubusercontent.com/3460638/81755897-f373fe00-946e-11ea-9a60-3f7e970866e8.png">

Reply:
Thanks for the screenshots - I can improve those error messages, filed:
https://github.com/brimsec/zq/issues/767